v2/manager.go

```diff
@@ -526,20 +526,19 @@ func (c *Manager) freeze(path string, state State) error {
 	}
 }
 
-func (c *Manager) MemoryEventFD() (uintptr, error) {
+func (c *Manager) MemoryEventFD() (int, uint32, error) {
```

Comment:
needs godoc

Reply:
> (int, uint32

why different types?

Reply:
Coz this is the base types for file descriptor and watch descriptor, at least `syscall.InotifyRmWatch()` expects it in that way.